pipelines/folioRancher/folioTestingTools/folioSchemaCompare/Jenkinsfile

```diff
@@ -0,0 +1,208 @@
+#!groovy
+import hudson.util.Secret
+import org.jenkinsci.plugins.workflow.libs.Library
+import java.time.*
+import org.folio.rest.GitHubUtility
+import org.folio.Constants
+import groovy.json.JsonSlurperClassic
+
+@Library('pipelines-shared-library@RANCHER-838') _
+
+properties([
+    buildDiscarder(logRotator(numToKeepStr: '20')),
+    disableConcurrentBuilds(),
+    parameters([
+        folioParameters.repository(),
+        folioParameters.branch('folio_branch_src', params.folio_repository),
+        folioParameters.branch('folio_branch_dst', params.folio_repository),
+        string(name: 'backup_name', defaultValue: '', description: '(Optional) RDS snapshot name. If empty create env from scratch', trim: true),
+        string(name: 'slackChannel', defaultValue: '', description: 'Slack channel name where send report (without #)', trim: true)])])
+        folioParameters.refreshParameters()
+
+def rancher_cluster_name = 'folio-perf'
+def rancher_project_name = 'data-migration'
+def config_type = 'performance'
+def tenant_id
+def admin_username
+def admin_password
+def tenant_id_clean ='clean'
+def diff = [:]
+def resultMap = [:]
+def pgadminURL = "https://${rancher_cluster_name}-${rancher_project_name}-pgadmin.ci.folio.org/"
+def okapiVersion = folioCommon.getOkapiVersion(params.folio_repository, params.folio_branch_src)
```

Comment:
Here better to use approach with eval of existing string function, please take a look on example here: https://github.com/folio-org/pipelines-shared-library/pull/322/files#diff-6c20ffb3189fdc0c2047408c17206ec7359df2c999b405732f779c6ba0b9326dR26